packages/react-inbox/src/types.ts

```diff
@@ -27,13 +26,18 @@ export interface InboxProps {
     onClick?: (event: React.MouseEvent) => void;
     onMouseEnter?: (event: React.MouseEvent) => void;
   }>;
+  renderBlocks?: {
```

Comment:
can someone send multiple text/action blocks? do we need that to be an array so it isn't a breaking change down the road?

Reply:
its a function that will be called for each block...